src/distilabel/llm/base.py

```diff
@@ -72,8 +70,8 @@ def generate(
         per input in `inputs`."""
         pass
 
-    @cached_property
-    def generate_parameters(self) -> List[inspect.Parameter]:
+    @property
```

Comment:
Then the `from functools import cached_property` can be removed too, right?

Reply:
yes! BTW I removed it because in some environments `pickle` was not able to serialize it.